assets/src/components/ui/styles/root/effects.css

```diff
@@ -0,0 +1,20 @@
+:root {
+
+    --ee-animation-spin: ee-animation-spin 1s linear infinite;
+
+}
+
+@-moz-keyframes ee-animation-spin {
+    100% { -moz-transform: rotate(360deg); }
+}
+
+@-webkit-keyframes ee-animation-spin {
+    100% { -webkit-transform: rotate(360deg); }
+}
+
+@keyframes ee-animation-spin {
+    100% {
+        -webkit-transform: rotate(360deg);
+        transform: rotate(360deg);
+    }
```

Comment:
you don't need to add browser specific rules because that is taken care of during the compile step.